src/generic-container/generic-container.ts

```diff
@@ -214,6 +214,10 @@ export class GenericContainer implements TestContainer {
     boundPorts: BoundPorts
   ): Promise<void>;
 
+  protected get hasExposedPorts(): boolean {
+    return Boolean(this.ports.length);
```

Comment:
This could be more explicit, instead of type coercion from number->boolean, this could just be `this.ports.length ===0`

Reply:
Sure, I can change that, but what's the advantage of doing it that way?

Reply:
> This could be more explicit, instead of type coercion from number->boolean
🙂

Reply:
Fair enough. Refactored in dfd591af969b814dde593b73d7abb87fdadec4c9